label.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_accessor :id, :title, :color, :items
+
+  def initialize(title, color)
+    @id = rand(1..9999)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    item.label = self unless @items.include?(item)
+    @items << item unless @items.include?(item)
+  end
+end
```

Comment:
Overall, the code appears to be well-structured and functional. It properly initializes the instance variables, provides a method to add items to the label, and prevents adding the same item multiple times.